include/tensor_filter_custom.h

```diff
@@ -99,9 +99,8 @@ typedef int (*NNS_custom_invoke) (void *private_data,
  * @brief Invoke the "main function". Without allocating output buffer. (fill in the given output buffer)
  * @param[in] private_data The pointer returned by NNStreamer_custom_init.
  * @param[in] prop GstTensorFilter's property values. Do not change its values.
- * @param[in] input pointer to input tensor, size = dim1 x dim2 x dim3 x dim4 x typesize, allocated by caller
+ * @param[in] input The array of input tensors, each tensor size = dim1 x dim2 x dim3 x dim4 x typesize, allocated by caller
  * @param[out] output The array of output tensors, each tensor size = dim1 x dim2 x dim3 x dim4 x typesize, the memory block for output tensor should be allocated. (data in GstTensorMemory)
- * @param[out] size The allocated size.
  * @return The output buffer allocated in the invoke function
```

Comment:
Needs one more change, this function returns integer
@return 0 if success

Reply:
OK. I will. :)